src/main/java/gridss/cmdline/MultipleSamFileCommandLineProgram.java

```diff
@@ -109,6 +119,57 @@ public List<SAMEvidenceSource> getSamEvidenceSources() {
 	    				getOffset(INPUT_MIN_FRAGMENT_SIZE, i, 0),
 	    				getOffset(INPUT_MAX_FRAGMENT_SIZE, i, 0)));
 	    	}
+			for (int i = 0; i < EMPTY_CATEGORIES.size(); i++) {
+				// in case we do not supply INPUT files, we can supply categories, and then we create an empty bam file for each category.
+				// in that way, the code can still run by there is no need to create empty bam file out of the code
+				File newFile = new File(WORKING_DIR,getOffset(EMPTY_CATEGORIES, i, null)+".bam");
+				File indexFile = new File(WORKING_DIR,getOffset(EMPTY_CATEGORIES, i, null)+".bam.bai");
+				// Create a SAM file header with the specified content of first line, RG and some SQ lines
+				SAMFileHeader header = new SAMFileHeader();
+				header.setAttribute("GO", "query");
```

Comment:
I am not 100% this is kosher. Why are you adding this?

Reply:
the header.setAttribute("GO", "query")?
I just need to fist line to look like valid bamfile. I will try to remove that, just copied from our current bam file, don't know what it says